cdci_data_analysis/flask_app/app.py

```diff
@@ -274,7 +274,8 @@ def run_analysis():
 @app.after_request
 def validate_schema(response):
     try:
-        QueryOutJSON().load(response.json)
+        if dispatcher_strict_validate:
+            QueryOutJSON().load(response.json)
```

Comment:
As I tried in #197 , checking if the response contains a json would make it work for the `download` functionalities

Reply:
it would let through various useless html too. Instead, we should check if response contains files, or, even better, if it is response to download.